src/main/java/com/github/zastrixarundell/toramsensei/Values.java

```diff
@@ -47,8 +47,19 @@
     public static void setDyeImages(List<String> dyeImages) { Values.dyeImages = dyeImages; }
 
     public static void setupJedis() {
-        URI redisURI = URI.create(System.getenv("REDIS_URL"));
-        jedis = new Jedis(redisURI);
+        try
+        {
+            URI redisURI = new URI(System.getenv("REDIS_URL"));
+            String password = redisURI.getUserInfo().split(":")[1];
```

Comment:
Breaking down this because apparently Jedis is getting an error when the username is specified. In the tests I've done, no username was needed to be specified. 

Should add logic to check whether the userinfo is present to you can just use `localhost` as the `REDIS_URL` and not get an array out of bounds error.